src/test/java/sirius/web/security/UserContextHelper.groovy

```diff
@@ -6,67 +6,57 @@
  * http://www.scireum.de - info@scireum.de
  */
 
-package sirius.web.security;
+package sirius.web.security
 
-import sirius.web.controller.Message;
-import sirius.web.http.TestResponse;
+import sirius.web.controller.Message
+import sirius.web.http.TestResponse
 
 /**
  * Provides boiler plate methods to assert a certain state in the {@link sirius.web.security.UserContext}.
  * <p>
  * This can be used to verify that no error messages where generated when using a template. This is probably used
  * in combination with {@link sirius.web.http.TestRequest}.
  */
-public class UserContextHelper {
+class UserContextHelper {
 
     private UserContextHelper() {
     }
 
     /**
-     * Expects that no messages where created at all
+     * Extracts the {@link sirius.web.security.UserContext} which was active when the response was built.
      *
-     * @return <tt>true</tt> if the assertion is fulfilled, <tt>false</tt> otherwise
+     * @param response the response to dissect
+     * @return the user context which was able while creating the response
      */
-    public static boolean expectNoMessages(TestResponse response) {
-        return getUserContext(response).getMessages().isEmpty();
+    public static UserContext getUserContext(TestResponse response) {
+        return response.getCallContext().get(UserContext.class)
     }
 
     /**
-     * Extracts the <tt>UserContext</tt> which was active when the response was built.
+     * Expects that no messages were created at all
      *
-     * @param response the response to dissect
-     * @return the user context which was able while creating the response
+     * @return <tt>true</tt> if the assertion is fulfilled, <tt>false</tt> otherwise
      */
-    public static UserContext getUserContext(TestResponse response) {
-        return response.getCallContext().get(UserContext.class);
+    public static void expectNoMessages(TestResponse response) {
+        assert getUserContext(response).getMessages().isEmpty()
     }
 
     /**
      * Expects that no error messages where created at all
      *
      * @return <tt>true</tt> if the assertion is fulfilled, <tt>false</tt> otherwise
      */
-    public static boolean expectNoErrorMessages(TestResponse response) {
-        for (Message msg : getUserContext(response).getMessages()) {
-            if (msg.getType() == Message.ERROR) {
-                return false;
-            }
-        }
-        return true;
+    public static void expectNoErrorMessages(TestResponse response) {
+        assert !getUserContext(response).getMessages().any { msg -> msg.getType() == Message.ERROR }
```

Comment:
I think `getUserContext(response).getMessages().all { msg -> msg.getType() != Message.ERROR }` would be easier to read